keyboards/xelus/kangaroo/rules.mk

```
@@ -5,16 +5,16 @@ MCU = STM32F072
 #   comment out to disable the options.
 #
 
-BOOTMAGIC_ENABLE = lite	# Virtual DIP switch configuration
-MOUSEKEY_ENABLE = yes	# Mouse keys
-EXTRAKEY_ENABLE = yes	# Audio control and System control
-CONSOLE_ENABLE = yes 	# Console for debug
-COMMAND_ENABLE = yes    # Commands for debug and configuration
-SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
-NKRO_ENABLE = yes	    # USB Nkey Rollover
-RGBLIGHT_ENABLE = no    # Enable keyboard RGB underglow
-BLUETOOTH_ENABLE = no   # Enable Bluetooth
-AUDIO_ENABLE = no       # Audio output
+BOOTMAGIC_ENABLE = lite	    # Virtual DIP switch configuration
```

I guessed I missed a tab on the previous suggestion.

```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
```